src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/OrganisationsController.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace NHSD.BuyingCatalogue.Organisations.Api.Controllers
 {
-    [Authorize]
+    [Authorize(Policy = "CanAccessOrganisation")]
```

Comment:
Could we make a static class for Policy names? The duplication of the name could get out of sync. Want to introduce one place to make a change.